mscore/musescore.cpp

```diff
@@ -6134,8 +6134,9 @@ void MuseScore::cmd(QAction* a, const QString& cmd)
                   }
             }
       else if (cmd == "edit-style") {
-            EditStyle es(cs, this);
-            es.exec();
+            if (!_styleDlg)
+                  _styleDlg = new EditStyle { cs, this };
+            _styleDlg->exec();
```

Comment:
Looking at this once more, it seems you have missed the `setScore()` call before executing the dialog (both here and in `ScoreView`). This makes style dialog edit a style of a wrong score, at least after it was created once. Could you please add the missing calls so I could merge this pull request?

Reply:
Done